tests/Java.Interop.Export-Tests/Java.Interop/ExportTest.cs

```diff
@@ -159,6 +159,66 @@ public static void StaticActionFloat (float f)
 				return false;
 			return true;
 		}
+
+		[JavaCallable ("staticManyParametersWithAutogeneratedDelegateType", Signature="(ZBCSIJFDLjava/lang/Object;Ljava/lang/String;Ljava/util/ArrayList;Ljava/lang/String;Ljava/lang/Object;DFJII)I")]
```

Comment:
I don't understand how this method differs from the existing "lots of parameters" test:

https://github.com/xamarin/java.interop/blob/03c227228955622511bafabe61a12e30e45dc6f7/tests/Java.Interop.Export-Tests/Java.Interop/ExportTest.cs#L109-L162

How does this (pair of) method(s) differ from the current `ExportType.staticFuncThisMethodTakesLotsOfParameters()` method?  They both have lots of parameters, and they both appear to have "similar" `[JavaCallable]` custom attribute values.  How does this test *differ*?